lib/components/app/desktop-nav.js

```diff
@@ -53,6 +54,7 @@ const DesktopNav = ({ otpConfig }) => {
 
         </Navbar.Brand>
       </Navbar.Header>
+      <ViewSwitcher sticky />
```

Comment:
I like where your heads at here, but I think we need to preserve this space for agency branding. Is there a possibility to stick this switcher at the bottom of the page (both mobile and desktop)? Before implementing this change, let's get consensus from the team.

Reply:
Are we expecting branding to get wider than this? Even wide-ish branding fits nicely. But agree good to discuss with everyone.

Reply:
As per discussion, it is now shown on desktop but mobile retains current behavior. Changed in 2b85bbc484d641323dcc885541d14ac5b7e03285